tools/ci_build/github/windows/helpers.ps1

```diff
@@ -381,9 +408,23 @@ function Install-Protobuf {
     }
     cd $protobuf_src_dir
     cd *
-    Get-Content $src_root\cmake\patches\protobuf\protobuf_cmake.patch | &'C:\Program Files\Git\usr\bin\patch.exe' --ignore-whitespace -p1
+	# Search patch.exe
```

Comment:
since we already utilize git command in build scripts, would use 'git apply' instead of 'patch' more reliable and less hassle?

Reply:
The patch is only needed for packaging pipelines, not for normal development. They are optional. You do not need to install  git at "C:\Program Files\Git". The updated powershell script in this PR will make the patch command optional.

Reply:
And all our CI machines, except the ones I don't have control, all have git installed at the same location: "C:\Program Files\Git"